server/src/test/java/io/crate/integrationtests/PgCatalogITest.java

```diff
@@ -328,4 +327,15 @@ public void test_pg_constrains_conkey_array_populated() throws Exception {
         ));
 
     }
+
+    @UseRandomizedSchema(random = false)
+    @Test
+    public void test_pg_class_oid_equals_cast_of_string_to_regclass() {
+        execute("CREATE TABLE persons (x INT)");
+        execute("SELECT " +
+            " '\"persons\"'::regclass::integer oid_from_relname, " +
```

Comment:
maybe it's better to remove the `::integer` cast so that in the future, we can easily catch and adjust the test, but as you like.